src/components/WishlistPanel/WishlistPanel.css

```diff
@@ -37,3 +37,97 @@
   max-width: 200px;
   min-width: 120px;
 }
+
+.product-wishList {
+  background: white;
```

Comment:
on the image from the trello board, the background is the same colour as the wishlist panel, so we can remove this